### DISCUSSION TITLE
PostsService.java#L186-L209

### DISCUSSION THREAD

**Discussion by zooneon** [dnd-side-project/dnd-5th-3-backend#132] [코드 리뷰]
https://github.com/dnd-side-project/dnd-5th-3-backend/blob/cb1f6c3d27c0978401e01b9eaf19f9a3634c2933/src/main/java/com/dnd5th3/dnd5th3backend/service/PostsService.java#L186-L209

@KangSeongHyo 
제가 mypage 관련 로직을 따로 분리하지 않고 PostsService에 작성했는데 따로 분리하는 것이 나을까요?

**Comment by KangSeongHyo** [comment]
개별적으로 관리할 필요가 있을경우라면 분리하면 좋을 것 같습니다.

**Reply by zooneon** [reply]
분리해보겠습니다 ㅎㅎ
